roles/uperf-bench/templates/workload.yml.j2

```diff
@@ -61,6 +61,13 @@ spec:
 {% endif %}
 {% if test_user is defined %}
              export test_user={{test_user}};
+{% endif %}
+{% if uperf.pin is sameas true %}
```

Comment:
This is fine for now, but moving forward we should be able to extract the node of the client and server through redis somehow.

Reply:
How do you propose we do this? As this would have to be queried from kube and provided to SNAFU via Redis.... There is not a *clean* way to provide this unless the user pins. Happy to hear what you think of.